packages/codept/codept.0.10.3/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+author: "octachron <octa@polychoron.fr>"
+maintainer: "octachron <octa@polychoron.fr>"
+homepage: "https://github.com/Octachron/codept"
+bug-reports: "https://github.com/Octachron/codept/issues"
+license: "gpl-3"
+dev-repo: "git+https://github.com/Octachron/codept.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@private-docs"] {with-doc}
```

Comment:
Are you sure you want to bundle the private docs to the documentation?

Reply:
Right, it is probably better to remove it for now.

Reply:
I have removed this line (and the odoc dependency).